arangod/Cluster/MaintenanceFeature.cpp

```diff
@@ -105,6 +109,39 @@ bool findNotDoneActions(std::shared_ptr<maintenance::Action> const& action) {
 
 }  // namespace
 
+arangodb::Result arangodb::maintenance::collectionCount(arangodb::LogicalCollection const& collection,
+                                                        uint64_t& c) {
+  std::string collectionName(collection.name());
+  transaction::StandaloneContext ctx(collection.vocbase());
+  SingleCollectionTransaction trx(
+    std::shared_ptr<transaction::Context>(
+      std::shared_ptr<transaction::Context>(), &ctx),
+    collectionName, AccessMode::Type::READ);
+
+  Result res = trx.begin();
+  if (res.fail()) {
+    LOG_TOPIC("5be16", ERR, Logger::MAINTENANCE) << "Failed to start count transaction: " << res;
```

Comment:
I suggest that this log and the next be `WARN` level

Reply:
Done.